src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -24,21 +29,19 @@ public static BufferedImage makeTextFrame(String item, int hBuffer, int vBuffer)
   }
   public static BufferedImage makeTextFrame(Color bg, Color frame, int hBuffer, int vBuffer)
   {
-    return makeTextMenu(bg, frame, bg, new ArrayList<String>(), 0, hBuffer, vBuffer);
+    return makeTextMenu(bg, frame, bg, new ArrayList<>(), 0, hBuffer, vBuffer);
```

Comment:
Are generic type arguments discouraged now?

Reply:
I was messing with the function signature while I was trying to get it to work, and not specifying is easier to "update".

...and it doesn't make any difference to the generated code. `ArrayList` is always of `Object`; the compiler just inserts downcasts for you.